notebooks/utils/utils.py

```diff
@@ -179,3 +179,30 @@ def timeseries_and_history_comparison(casename, output_roots):
         else:
             print(f"Could not find time series for all variables in year {year:04}")
         print("----")
+
+
+################################################################################
+
+
+def dict_copy_vals(src, dst, keys):
+    for key in keys if type(keys) == list else [keys]:
+        if key in src:
+            dst[key] = src[key]
+
+
+################################################################################
+
+
+def print_key_metadata(ds, msg=None):
+    print(64 * "*")
+    if msg is not None:
+        print(msg)
+    print(64 * "*")
```

Comment:
Extremely nit-picky, but I think you want to indent this print statement to be part of the `if msg is not None` block; currently the output options (using 4 `*` instead of 64) are

```
****
msg is not None
****
ds.chunks
[value]
```

or

```
****
****
ds.chunks
[value]
```

And this change would make the latter (when `msg is None`)

```
****
ds.chunks
[value]
```

Reply:
good catch, I'll indent that print statement